test/registries.test.js

```diff
@@ -0,0 +1,48 @@
+describe('Registry', function () {
+  describe('one per app', function () {
+    it('should allow two apps to reuse the same model name', function (done) {
+      var appFoo = loopback();
+      var appBar = loopback();
+      var modelName = 'MyModel';
+      var subModelName = 'Sub' + modelName;
```

Comment:
I was expecting that `subModel` is a subclass of `Model`. Can you use a different prefix please, e.g. `relatedModelName`? Or perhaps be more specific and use `Product` and `Category` instead of `Model` and `subModel`.